source/server/proto_descriptors.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+namespace Envoy {
+namespace Server {
+bool validateProtoDescriptors();
```

Comment:
Can you add a brief comment here on what this does? Maybe just move the comment from the function definition.